packages/datadog-plugin-sequelize/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const {
+  sandboxCwd, useSandbox, varySandbox, curl,
+  FakeAgent, spawnPluginIntegrationTestProc
+} = require('../../../../integration-tests/helpers')
+const { assert } = require('chai')
+const { withVersions } = require('../../../dd-trace/test/setup/mocha')
+
+withVersions('sequelize', 'sequelize', '>=4', version => {
```

Comment:
Do we need to limit the version?

The same question about other test files with version limits.